charts/tekton-apps/templates/_snippets.tpl

```diff
@@ -215,3 +214,10 @@ the way you use it:
   value: {{ .default }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Check what namespace need to use, from project or component or extraApp field
```

Comment:
Not actual docs

```suggestion
Get namespace and return component `destinationNamespace` if it exists, otherwise return project's `namespace`.
```